manifests/0000_50_olm_00-namespace.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
```

Comment:
@ecordell i'm going to let the CVO keep creating this one, so we don't need to exclude it.